src/deepsysid/pipeline/configuration.py

```diff
@@ -270,6 +299,22 @@ def initialize_model(
     return model
 
 
+def initialize_tracker(
+    experiment_config: ExperimentConfiguration,
+) -> Callable[[EventData], None]:
+    def tracker_fcn(_):
+        return None
+
+    tracker = tracker_fcn
+    if experiment_config.tracker is not None:
+        trackers: List[BaseEventTracker] = list()
+        for tracker_config in experiment_config.tracker.values():
+            tracker_class = retrieve_tracker_class(tracker_config.tracking_class)
+            trackers.append(tracker_class(tracker_config.parameters))  # type: ignore
```

Comment:
@AlexandraBaier without `type: ignore` mypy complains about too many inputs, do you know why this is happening?